datacommons/utils.py

```diff
@@ -138,7 +138,7 @@ def clean_frame(pd_frame):
 # ------------------------- INTERNAL HELPER FUNCTIONS -------------------------
 
 
-def _send_request(req_url, req_json={}, compress=False):
+def _send_request(req_url, req_json={}, compress=False, post=True):
```

Comment:
Oh, interesting -- why are there only GET calls for popops and placekml?

Reply:
get_pop_obs and place_kml only requires a single param (dcid). The other api needs POST as they need to accept multiple parameters that include list, otherwise we can just use GET for those.